CHANGELOG.md

```diff
@@ -95,6 +96,7 @@
  3. Design changes:
  - Split features into API modules for each hardware variant
  - Fixed/updated/added CLI commands
+ 4. HPE Synergy Image Streamer Support:
```

Comment:
Sure! You can always just say we added the necessary design changes to support the resources in the future.

And I know some of the other PRs have added a new section with an actual version (ie 3.1.0),  but since we don't know the release version of these new changes yet, I'd add them to a section titled `Unreleased Changes` or something similar.